app/scripts/services/dimItemService.factory.js

```diff
@@ -464,12 +464,23 @@
       }
     }
 
+    function isClassified(item) {
+        if (!item.action) {
```

Comment:
Rather than doing this check in dimItemService, why not just properly set `classified` when we build the item in `dimStoreService`?